### PR TITLE
LF-3055: Replace Unit component for % with Input component in WaterUsageCalculatorModal

### PR DESCRIPTION
**Pull request by SayakaOno**
**Description**

- Replace `<Unit />` used for `% with Input component` with `<Input />` in WaterUsageCalculatorModal. (The unit component is not designed to be used for percentage)
- pass `errors` to `WaterUsageCalculatorModal` from its parent component.
- delete `percentage_location` object which should not be used.

Note: You will see a warning in the console when you open the water use calculator modal, and it is going to be addressed in this [ticket](https://lite-farm.atlassian.net/browse/LF-3080). (it is not related to the input component)

Jira link: https://lite-farm.atlassian.net/browse/LF-3055

**Type of change**

- [x] Bug fix (non-breaking change which fixes an issue)

**How Has This Been Tested?**

- [x] UI components visually reviewed on desktop view

1. go to /tasks
2. "+ Create a task"
3. Select “Irrigate”
4. "Continue" until you see "Tell us about the Irrigation work task"
5. Select "Depth" for “How do you measure water use for this irrigation type?”
6. Click “Calculate Water Usage”
7. the calculator should look and work as before

**Checklist:**

- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] The precommit and linting ran successfully
- [ ] I have added or updated language tags for text that's part of the UI
- [ ] I have added "MISSING" for all new language tags to languages I don't speak
- [ ] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

### CODE REVIEW

File: packages/webapp/src/components/Modals/WaterUsageCalculatorModal/index.jsx [hunk 1]
```diff
@@ -6,11 +6,11 @@ import styles from '../QuickAssignModal/styles.module.scss';
 import Button from '../../Form/Button';
 import PropTypes from 'prop-types';
 import Unit, { getUnitOptionMap } from '../../Form/Unit';
+import Input, { getInputErrors } from '../../Form/Input';
```
Comment (Duncan-Brain) [inline packages/webapp/src/components/Modals/WaterUsageCalculatorModal/index.jsx:9]: Just double importing /Form/input here.

**Comment by Duncan-Brain** [comment]
@SayakaOno 

Looks good, left just one inline comment.

File: packages/webapp/src/components/Modals/WaterUsageCalculatorModal/index.jsx (hunk 1 above)
Comment (SayakaOno) [reply packages/webapp/src/components/Modals/WaterUsageCalculatorModal/index.jsx:9]: Oh! Thank you!

**Review by Duncan-Brain — APPROVED** [review]

**Review by kathyavini — APPROVED** [review]
Sorry for confusing the scope of this ticket with the other 🙂 It looks good! I'll model the error handling + validations for the % Input Broadcast Plan of this pattern.

**Pull request merged**